### PR TITLE
fix(client): parse logs as JSON string

### PR DESCRIPTION
**Pull request by olalonde**
Fixes issue #4657.

### CODE REVIEW

File: client/Makefile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 include ../includes.mk
 
+.PHONY: build
```
Comment (olalonde) [inline client/Makefile]: not sure if there's something wrong with my environment but `make build` wouldn't work without that

Comment (Joshua-Anderson) [inline client/Makefile]: That's a reminant from the build folder left over from the old python client.

**Comment by olalonde** [comment]
Any chance to get this merged? Should I removed the `.PHONY` line?

File: client/Makefile (hunk 1 above)
Comment (bacongobbler) [inline client/Makefile]: let's remove this. Just remove the `client/build` folder locally and everything should work fine again. :)

**Comment by bacongobbler** [comment]
@olalonde there's a server-side fix available in #5004 which fixes the underlying issue. With that fix logs are now properly formatted.

**Pull request closed without merging**